### DISCUSSION TITLE
匹配规则会自主休眠么？

### DISCUSSION THREAD

**Discussion by MagicHello** [gkd-kit/gkd#812] [Announcements]
### 新功能描述

我最近在做一个15秒广告自动跳过，我限定了界面id并match了两条：已观看完成和跳过（理想状态是识别已观看完成后执行跳过按钮），但我在调试时发现，无法执行，我将（已观看完成）匹配更换为（视频剩余12，13，14，15秒），发现均可以匹配到，但是之后，我的规则再也无法匹配，我也没有做任何的延迟（包括matchtime等），这是gkd自带的休眠机制么？（当然目前主流的15秒广告跳过用的都是actionDelay，但是在网络存在延迟的情况下很容易误触）

**Comment by MagicHello** [comment]
希望可以有一个能够解决这个规则自主休眠的办法

**Comment by MagicHello** [comment]
由于不清楚是特性还是bug，只能放在feature了🥵

**Comment by lisonge** [comment]
你可以使用 https://github.com/gkd-kit/gkd/discussions 没有必要使用 issue

**Comment by lisonge** [comment]
你可以尝试使用 https://gkd.li/api/interfaces/RawCommonProps#forcedtime

**Reply by lisonge** [reply]
规则没有自主休眠的说法，能不能运行全看界面有没有无障碍事件发出

否则就使用我上面给的属性

**Reply by MagicHello** [reply]
感谢，明日我尝试一下（今天广告看完了）😍
